cmd/daytona/config/ssh_file.go

```diff
@@ -15,11 +15,19 @@ import (
 var sshHomeDir string
 
 func ensureSshFilesLinked() error {
-	// Make sure ~/.ssh/config file exists
+	// Make sure ~/.ssh/config file exists if not create it
 	sshDir := filepath.Join(sshHomeDir, ".ssh")
 	configPath := filepath.Join(sshDir, "config")
 
-	_, err := os.Stat(configPath)
+	_, err := os.Stat(sshDir)
```

Comment:
@ezhil56x no need to check specifically for the ssh dir.

I would suggest just doing `os.MkdirAll(sshDir, 0700)` before writing to the `configPath`. MkdirAll will be a noop if the dir does exist.

Reply:
Done with the changes